### PR TITLE
Prefixed credentials

### PR DESCRIPTION
**Pull request by EdgeCaseBerg**
As originally discussed [here](https://github.com/Rhinofly/play-aws-utils/issues/8)

Ability to use multiple credentials in a single application easily. 

I can add examples to the README file, but wanted to get feedback on the
implementation itself first.

### CODE REVIEW

File: src/main/scala/fly/play/aws/AwsCredentials.scala [hunk 1]
```diff
@@ -18,12 +18,14 @@ object AwsCredentials extends ((String, String, Option[String]) => AwsCredential
   def apply(accessKeyId: String, secretKey: String, token: Option[String] = None): AwsCredentials =
     SimpleAwsCredentials(accessKeyId, secretKey, token)
 
-  implicit def fromConfiguration(implicit app: Application): AwsCredentials = {
+  implicit def fromConfiguration(implicit app: Application): AwsCredentials = fromConfiguration("aws")
+
+  implicit def fromConfiguration(prefix: String = "aws")(implicit app: Application): AwsCredentials = {
```
Comment (EECOLOR) [inline src/main/scala/fly/play/aws/AwsCredentials.scala]: This one does not need to be implicit. I also think the default value is unneeded.

What do you think about:

```
def fromConfiguration(prefix: String)(implicit app: Application): AwsCredentials
```

Comment (EdgeCaseBerg) [inline src/main/scala/fly/play/aws/AwsCredentials.scala]: I think you're right about it not needing to be an implicit and the default isn't necessary as well. I'll update the code.

Comment (EdgeCaseBerg) [inline src/main/scala/fly/play/aws/AwsCredentials.scala]: @EECOLOR 
Updated and pushed the new commit into the pull request.

**Comment by EECOLOR** [comment]
Yesterday I have worked on the 6.0.0 branch and force pushed it. So it might be a good idea to rebase on that. If you don't know how that works it's fine. As soon as I am able to spend more time I will integrate this change.

**Comment by EdgeCaseBerg** [comment]
I've pulled down your force-pushed 6.0.0 branch. Saved my old branch to a different name, branched off of the new 6.0.0 branch, and cherry picked the commits with the prefix credentials feature. Then force pushed my own prefixed-credentials branch so that it shares the common base with your new 6.0.0.

**Comment by EECOLOR** [comment]
Awesome.

**Comment by EECOLOR** [comment]
I have intergrated your result in the 6.0.0 version

**Pull request closed without merging**